### PR TITLE
N21-2019 show media shelf tool usages

### PR DESCRIPTION
**Pull request by IgorCapCoder**
# Short Description
when school admin deletes a school external tool, the tool usages from media shelf are also shown

## Links to Ticket and related Pull-Requests
- https://n21-2019-show-tool-usage.nbc.dbildungscloud.dev/administration/school-settings
- https://github.com/hpi-schul-cloud/schulcloud-server/pull/5071
- https://github.com/hpi-schul-cloud/superhero-dashboard/pull/257

## Checklist before merging

- [x] QA: In addition to review, the code has been manually tested (if manual testing is possible)
- [x] PO: Any deviation from requirements was agreed with Product-Owner / ticket author / support-team
- [x] DEV: Every new component is implemented having accessibility in mind (e.g. aria-label, role property)

> Notice: Please keep this Pull-Request as a Draft (or add WIP label), until it is ready to be reviewed

### CODE REVIEW

File: src/components/administration/ExternalToolSection.vue [hunk 1]
```diff
@@ -69,20 +69,68 @@
...
+								{
 									courseCount: metadata.course,
+								}
+							)
+						"
+						component="p"
+					/>
+					<RenderHTML
+						data-testid="delete-dialog-content-board-elements"
+						:class="isMediaBoardUsageVisible ? 'text-md mb-0' : 'text-md'"
```
Comment (IgorCapCoder) [inline src/components/administration/ExternalToolSection.vue:99]: 😎

File: src/components/administration/ExternalToolSection.vue [hunk 2]
```diff
@@ -236,6 +289,14 @@ export default defineComponent({
 			isDeleteDialogOpen.value = false;
 		};
 
+		const isMediaBoardUsageVisible: ComputedRef<boolean> = computed(() => {
+			if (!metadata.value) return false;
```
Comment (arnegns) [inline src/components/administration/ExternalToolSection.vue:293]: Please use brackets here

File: src/components/administration/ExternalToolSection.vue [hunk 3]
```diff
@@ -236,6 +289,14 @@ export default defineComponent({
...
+			return (
```
Comment (arnegns) [inline src/components/administration/ExternalToolSection.vue:294]: Please dont use inline returns. Just return a variable

File: src/locales/de.ts [hunk 4]
```diff
@@ -243,8 +243,16 @@ export default {
 		'Die schulspezifischen Parameter für das externe Tool werden hier konfiguriert. Nach dem Speichern der Konfiguration ist das Tool innerhalb der Schule verfügbar.<br><br>\nWeitere Informationen sind in unserem <a href="https://docs.dbildungscloud.de/x/uoKqDg" target="_blank">Hilfebereich zu externen Tools</a> zu finden.',
 	"components.administration.externalToolsSection.dialog.content.metadata.error":
 		"Die Verwendung des Tools konnte nicht ermittelt werden.",
-	"components.administration.externalToolsSection.dialog.content":
-		"Sind Sie sich sicher, dass Sie das Tool <b>{itemName}</b> löschen wollen? </br></br>Zurzeit wird das Tool wie folgt genutzt: <br><b>{courseCount}</b> Kurs(e) <br><b>{boardElementCount}</b> Spalten-Board(s) <br><br> Achtung: Wenn das Tool entfernt wird, kann es für diese Schule nicht mehr genutzt werden.",
+	"components.administration.externalToolsSection.dialog.content.header":
```
Comment (arnegns) [inline src/locales/de.ts:246]: nice

File: src/components/administration/ExternalToolSection.vue (hunk 1 above)
Comment (arnegns) [inline src/components/administration/ExternalToolSection.vue:99]: I am missing a test here :)

File: tests/test-utils/factory/schoolExternalToolMetadataFactory.ts [hunk 5]
```diff
@@ -1,8 +1,9 @@
-import { Factory } from "fishery";
 import { SchoolExternalToolMetadata } from "@/store/external-tool";
+import { Factory } from "fishery";
```
Comment (arnegns) [inline tests/test-utils/factory/schoolExternalToolMetadataFactory.ts:2]: Please check your import order :)

**Review by arnegns — CHANGES_REQUESTED** [review]

File: src/components/administration/ExternalToolSection.vue (hunk 2 above)
Comment (GordonNicky) [reply src/components/administration/ExternalToolSection.vue]: added in [ce5a21f](https://github.com/hpi-schul-cloud/nuxt-client/pull/3298/commits/ce5a21fba1686cf081dd68e76ac17f7fa8526228)

File: src/components/administration/ExternalToolSection.vue (hunk 3 above)
Comment (GordonNicky) [reply src/components/administration/ExternalToolSection.vue]: changed in [ce5a21f](https://github.com/hpi-schul-cloud/nuxt-client/pull/3298/commits/ce5a21fba1686cf081dd68e76ac17f7fa8526228)

File: tests/test-utils/factory/schoolExternalToolMetadataFactory.ts (hunk 5 above)
Comment (GordonNicky) [reply tests/test-utils/factory/schoolExternalToolMetadataFactory.ts]: order swapped in [ce5a21f](https://github.com/hpi-schul-cloud/nuxt-client/pull/3298/commits/ce5a21fba1686cf081dd68e76ac17f7fa8526228) :)

**Review by arnegns — APPROVED** [review]

File: src/components/administration/ExternalToolSection.vue (hunk 1 above)
Comment (GordonNicky) [reply src/components/administration/ExternalToolSection.vue:98]: Implemented in [ce5a21f](https://github.com/hpi-schul-cloud/nuxt-client/pull/3298/commits/ce5a21fba1686cf081dd68e76ac17f7fa8526228), I hope it is not too much

**Pull request merged**